pkg/ccl/backupccl/restore_planning.go

```diff
@@ -2092,6 +2143,32 @@ func doRestorePlan(
 	return sj.ReportExecutionResults(ctx, resultsCh)
 }
 
+// renameTargetDatabase updates the name in the database descriptors if the user specified a
+// new_db_name in their single database restore
+func renameTargetDatabase(
```

Comment:
nit: renameTargetDatabaseDescriptor

I'd also like to see a little more in this comment:

renameTargetDatabaseDescriptor updates the name in the target database descriptor to the user specified `new_db_name`. We update the database descriptor in both `sqlDescs` that contains all the descriptors being restored, and `restoreDBs` that contains just the target database descriptor. Renaming the target descriptor ensures accurate name resolution during restore.

Reply:
done.